app/vmstorage/main.go

```diff
@@ -67,6 +67,8 @@ var (
 		"See https://docs.victoriametrics.com/single-server-victoriametrics/#cache-tuning")
 	cacheSizeIndexDBDataBlocks = flagutil.NewBytes("storage.cacheSizeIndexDBDataBlocks", 0, "Overrides max size for indexdb/dataBlocks cache. "+
 		"See https://docs.victoriametrics.com/single-server-victoriametrics/#cache-tuning")
+	cacheSizeIndexDBDataBlocksSparse = flagutil.NewBytes("storage.cacheSizeIndexDBDataBlocksSparse", 0, "Overrides max size for indexdb/dataBlocksSparse cache. "+
```

Comment:
If this can be overriden, it should have been mentioned in https://docs.victoriametrics.com/#cache-tuning
I am ok with not having it overridable, though. As it seems too complicated to explain and shoulb be something the system adjusts on its own.

Reply:
As far as I can see there is explicit mentions for any caches at this page, only generic guidance about cache tuning. Should we mention this one specifically?

Reply:
Yeah, my bad. Quickly glanced the page without actually reading it(